apps/console/src/features/claims/components/wizard/local-claim/basic-details-local-claims.tsx

```diff
@@ -18,9 +18,13 @@
 
 import { TestableComponentInterface } from "@wso2is/core/models";
 import { Field, FormValue, Forms } from "@wso2is/forms";
+import { GenericIcon, Hint } from "@wso2is/react-components";
+import { InlineEditInput } from "@wso2is/react-components/src";
```

Comment:
Shall we move this to `"@wso2is/react-components"`

Reply:
Fixed with 9ead70e